src/test/acceptance/features/confirmation-code-navigation.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Confirmation code navigation
+  In order to apply for the HTBHF programme
+  As a potential claimant
+  Once I have entered my confirmation code
+  If I change the contact details used to receive that code
+  I have to go through the send and enter code part of the application to receive a new code.
+
+  Background:
+    Given I am on the first page of the application
+
+  Scenario: Changing my phone number after using it to receive a code requires me to be sent a new code
+    When I complete the application with valid details, selecting to receive my confirmation code via text
+    And I choose to change my answer to phone number
+    And I enter in a new phone number
+    And I must continue the application from the email address page, including entering a new code
+
+  Scenario: Changing my email address after using it to receive a code requires me to be sent a new code
+    When I complete the application with valid details, selecting to receive my confirmation code via email
+    And I choose to change my answer to email address
+    And I enter in a new email address
+    And I must continue the application from the send code page, including entering a new code
```

Comment:
What about `I am asked to enter a new code`? Same above too.

Reply:
Unfortunately the two steps go to different pages, the scenario above requires an additional step

Reply:
Ok, but they can still both be reworded.

Reply:
Updated to `Then I must complete all steps after the email address page, including entering a new code`